src/pages/home/report/ReportActionItem.js

```diff
@@ -239,6 +253,13 @@ class ReportActionItem extends Component {
         if (this.props.action.actionName === CONST.REPORT.ACTIONS.TYPE.CHRONOSOOOLIST) {
             return <ChronosOOOListActions action={this.props.action} reportID={this.props.report.reportID} />;
         }
+
+        const whisperedTo = lodashGet(this.props.action, 'whisperedTo', []);
+        const isWhisper = _.size(whisperedTo) > 0;
+        const isMultipleParticipant = _.size(whisperedTo) > 1;
```

Comment:
`_.size()` is usually only used for objects, not arrays where you can use length.
```suggestion
        const isWhisper = whisperedTo.length > 0;
        const isMultipleParticipant = whisperedTo.length > 1;
```